src/gsy_e/models/area/__init__.py

```diff
@@ -252,6 +252,19 @@ def get_results_dict(self):
         return {}
 
 
+class CoefficientAreaException(Exception):
+    """Exception that is raised when serializing an area."""
+
+
+def assert_coefficient_area_settings(
```

Comment:
I'd suggest to use a different naming in place of "assert". Since `assert` has a very specific meaning in Python, this might be confusing. Maybe `validate_coefficient_area_setting()` could be an option?

Reply:
Ok, will do